src/Microsoft.DotNet.Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <PackageId>Microsoft.DotNet.Web.ProjectTemplates.2.1</PackageId>
     <NuspecFile>Microsoft.DotNet.Web.ProjectTemplates.2.1.nuspec</NuspecFile>
```

Comment:
Could we also remove the version number from the nuspec filename (so future version updates will be less noisy)? Or does it have to match the PackageId?

Reply:
Good catch. File name can be version-less. I just missed this one. I'll update it.